src/test/java/mate/academy/bot/telegrambot/TelegramBotApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.bot.telegrambot;
```

Comment:
remove this file?

Reply:
this is standard SpringBoot smoke test that verifies the app is running OK